src/tests/test_util.h

```diff
@@ -23,15 +22,15 @@
   }
 #define ASSERT(expr)                                                                       \
   if (!(expr)) {                                                                           \
-    fprintf(stderr, "%s:%d: Assertion '%s' Failed\n", __FILE__, __LINE__, __STRING(expr)); \
+    fprintf(stderr, "%s:%d Assertion '%s' Failed\n", __FILE__, __LINE__, __STRING(expr));  \
     return -1;                                                                             \
   }
 
 #define ASSERT_STRING_EQ(s1, s2) ASSERT(!strcmp(s1, s2));
 
-#define ASSERT_EQUAL_INT(x, y, ...)                                 \
+#define ASSERT_EQUAL(x, y, ...)                                 \
   if (x != y) {                                                     \
-    fprintf(stderr, "%s:%d: ", __FILE__, __LINE__);                 \
+    fprintf(stderr, "%s:%d: Assertion Failed " __VA_ARGS__ ": ", __FILE__, __LINE__);                 \
```

Comment:
`%f` and `(double)` sounds good to me, should work well. I don't think any runes will be cut off, because the max value for utf32 and utf16 is 0x10ffff as far as I can tell.